### DISCUSSION TITLE
Suppress message parameter

### DISCUSSION THREAD

**Discussion by pasqualeceglie** [olahallengren/sql-server-maintenance-solution#457] [Ideas] [answered]
Hi Ola, do you think it is a good idea to have a parameter to suppress info messages (RAISERROR with severity 10)?

**Comment by olahallengren** [comment, marked as answer]
The drawback with a parameter like that, is that it would be very difficult to find out why the script behaved the way it did. For example why a database was skipped, why the backup type was changed or why there was an error. The informational messages are invaluable in understanding what happened.
